webapp-django/crashstats/crashstats/static/crashstats/js/socorro/topcrash.js

```diff
@@ -15,23 +15,25 @@ var Correlations = (function() {
     // a hash table where we keep all unique signatures that have correlations
     var all_signatures = {};
 
-    function loadCorrelations(type, callback) {
-        var url = SocReport.sig_base + '?correlation_report_type=' + type +
-                  '&' + SocReport.path + '&platforms=';
+    function loadCorrelations(types, callback) {
         var data = {
-            correlation_report_type: type,
+            correlation_report_types: types,
             product: SocReport.product,
-            version: SocReport.version
+            version: SocReport.version,
+            platforms: [],
         };
-        // osnames is a dictionary so...
-        data.platforms = $.map(osnames, function(i, osname) {
-            return osname;
-        }).join(',');
-
-        $.getJSON(SocReport.sig_base, data, function (json) {
-            $.each(json.hits, function(i, sig) {
-                all_signatures[sig] = 1;
-            });
+        for (var platform in osnames) {
+            data.platforms.push(platform);
+        }
+        $.getJSON(SocReport.sig_base + '?' + $.param(data, true))
```

Comment:
Apparently you can do things like this with `$.getJSON(url, {foo: 'bar'})` but not this: `$.getJSON(url, {foo: ['bar', 'baz']})`. That's why I use `$.param(data, true)`. It then turns it into `'foo=bar&foo=baz'`